### DISCUSSION TITLE
is pockethost down?

### DISCUSSION THREAD

**Discussion by FrenchFryInteractive** [pockethost/pockethost#254] [Q&A] [answered]
Unexpected response {"url":"","status":0,"data":{},"isAbort":false,"originalError":{"cause":{"errno":-111,"code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":8090}},"name":"ClientResponseError 0"} from mothership

This is message I get, when trying to refresh.

**Comment by benallfree** [comment, marked as answer]
Sounds related to #253. Moving both of these to #223
